### PR TITLE
UI: Added hostname to LogBox title

### PR DESCRIPTION
**Pull request by unkwntech**
I added the hostname to the title bar of the log/tail window to indicate which server the script that is being viewed is running on.

Screenshot of log window showing the hostname "home" displayed as a prefix in the title bar.

### CODE REVIEW

File: src/ui/React/LogBoxManager.tsx [hunk 1]
```diff
@@ -249,7 +249,12 @@ function LogWindow(props: IProps): React.ReactElement {
 
   function title(full = false): string {
     const maxLength = 30;
-    const t = `${script.filename} ${script.args.map((x: ScriptArg): string => `${x}`).join(" ")}`;
+    const server = GetServer(script.server);
```
Comment (Snarling) [inline src/ui/React/LogBoxManager.tsx]: script.server here is already a hostname, is there a reason you're doing another lookup to fetch the actual server using that name?

File: src/ui/React/LogBoxManager.tsx [hunk 2]
```diff
@@ -249,7 +249,12 @@ function LogWindow(props: IProps): React.ReactElement {
...
+    let hostname = "";
+    if (server !== null) {
+      hostname = `${server.hostname}: `;
+    }
+    const t = `${hostname}${script.filename} ${script.args.map((x: ScriptArg): string => `${x}`).join(" ")}`;
```
Comment (Snarling) [inline src/ui/React/LogBoxManager.tsx]:
```suggestion
    const t = `${script.server}: ${script.filename} ${script.args.join(" ")}`;
```
It's pre-existing code, but that map function to turn arguments into strings is also not needed - [joining them](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join#description) already does that.

**Review by Snarling — CHANGES_REQUESTED** [review]

**Review by Snarling — APPROVED** [review]

**Comment by brunnerh** [comment]
I feel like this should be an option, my titles already don't fit fairly often.
image

**Comment by Snarling** [comment]
Yeah I can see how that would be a problem. I'll open an issue for it.

**Comment by d0sboots** [comment]
I am undoing this in #508, but it adds the ability to both get the current title and set a custom title, so hopefully that makes it easy to reimplement for the scripts where you want it.

**Pull request merged**